cspatients/tests.py

```diff
@@ -312,6 +312,77 @@ def test_get_view(self):
             template_name="cspatients/view.html"
         )
 
+
+class PatientViewTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(
+            username="Itai"
+        )
+        self.client.force_login(self.user)
+
+    def test_view_non_existing_patient_returns_404(self):
+        response = self.client.get("/cspatients/patient/SHFLFLF")
+        self.assertEqual(response.status_code, 404)
+        self.assertTemplateUsed(
+            response=response,
+            template_name="cspatients/patient.html"
+        )
+        self.assertInHTML(
+            "Sorry. No patient such patient was found.",
+            str(response.content)
+        )
+
+    def test_view_one_existing_patiententry(self):
+        jane = Patient.objects.create(
+            name="Jane",
+            patient_id="XXXXXX",
+            age=10
+        )
+
+        PatientEntry.objects.create(
+            patient_id=jane,
+            urgency=2,
+            decision_time=timezone.now()
+        )
+
+        response = self.client.get("/cspatients/patient/XXXXXX")
```

Comment:
I might be getting a bit nit picky here, but I think this code might be improved by doing the following:
`response = self.client.get("/cspatients/patient/{}".format(jane.patient_id))`

This helps to do 2 things:
- makes things slightly more DRY
- communicates a bit more clearly what the XXXXXX would represent

I'll leave this one up to you though.

Reply:
@nathanbegbie I think I will follow the suggestion. Makes much more sense to be clear.